### PR TITLE
6363 add test harness

### PR DESCRIPTION
**Pull request by alibombaye**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  "extends": "airbnb-base",
+  "env": {
+    "browser": true,
+    "mocha": true,
```
Comment (kat1906) [inline .eslintrc.js:5]: do we even use mocha?

**Comment by kat1906** [comment]
which app(s) have you tested this with?

File: .eslintrc.js (hunk 1 above)
Comment (alibombaye) [reply .eslintrc.js:5]: No we don't. I'll get it removed. We need to do the same with the other apps too

**Comment by alibombaye** [comment]
I've tested this with PB. I'll get a PR raised for that too.

**Pull request merged**